scripts/protopackage.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/sh
+set -eo pipefail
+
+# This script appends the "option go_package" proto option to the file located at $FNAME.
+# This option is not included in the proto files to allow the files to more easily
+# be hosted in github.com/tendermint/spec and shared between other repos.
+# This option specifies what the package will be named when imported by other packages.
+
+FNAME=$1
```

Comment:
```suggestion
FNAME="${1:?missing required .proto filename}"
```

(or include `-u` in the options, or maybe even both)

Reply:
Added the :? syntax. This works better since I'm now relying on being able to check if $3 is set or not for import path overriding.